pkg/apis/core/types.go

```diff
@@ -1807,6 +1807,13 @@ type ExecAction struct {
 	Command []string
 }
 
+// SleepAction describes a sleep action.
+type SleepAction struct {
+	// The sleep duration in seconds.
```

Comment:
needs comments about bounds

Reply:
Why is it optional?  Can we imagine a Sleep struct without it?

Reply:
Most of the other fields are optional, I guessed because it goes into the protobuf and protobuf has this big warning about `Required Is Forever`.
But then again I can't imagine it to be missing, I'll drop it.